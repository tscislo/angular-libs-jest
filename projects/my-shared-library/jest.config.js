const jestConfig = require('../../unit-tests/parent-jest.config')

module.exports = {
  "globals": {
    "ts-jest": {
      "tsConfigFile": "projects/my-shared-library/tsconfig.spec.json"
    }
  },
  ...jestConfig
}
