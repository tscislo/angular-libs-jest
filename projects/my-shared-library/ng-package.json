{
  "$schema": "../../node_modules/ng-packagr/ng-package.schema.json",
  "dest": "../../dist/my-shared-library",
  "deleteDestPath": false,
  "lib": {
    "entryFile": "src/public_api.ts"
  }
}