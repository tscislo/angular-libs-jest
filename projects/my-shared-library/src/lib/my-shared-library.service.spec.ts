import { TestBed, inject } from '@angular/core/testing';

import { MySharedLibraryService } from './my-shared-library.service';

describe('MySharedLibraryService', () => {
  beforeEach(() => {
    TestBed.configureTestingModule({
      providers: [MySharedLibraryService]
    });
  });

  it('should be created', inject([MySharedLibraryService], (service: MySharedLibraryService) => {
    expect(service).toBeTruthy();
  }));
});
