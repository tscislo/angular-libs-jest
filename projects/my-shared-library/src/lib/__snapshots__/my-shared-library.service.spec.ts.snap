// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MySharedLibraryService should getCountriesWithPopulationGrater get countries with population grater then given 1`] = `
Array [
  Object {
    "name": "Ireland",
    "population": 5,
  },
]
`;
